### PR TITLE
[ISSUE-656] Support SLES 15 SP3 in CSI

### LINKED ISSUE #656: Support SLES 15 SP3 in CSI
**Is your feature request related to a problem? Please describe.**
Volume creation on SLES 15 SP3 takes more than 10 minutes. After using node-kernel-5.4 image problem went away.

**Describe the solution you'd like**
Should replace hanging tools, like parted and partprobe

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Aber4Nod**
### Resolves #656 

Replace parted with sgdisk
Replace partprobe with blockdev
Turn off lvm synchronization with udev

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Provide test details_

### CODE REVIEW

File: pkg/node/Dockerfile [hunk 1]
```diff
@@ -6,4 +6,10 @@ ADD     node  node
 
 EXPOSE  9999
 
+RUN sed -i \
+        -e 's/udev_sync = 1/udev_sync = 0/' \
+        -e 's/udev_rules = 1/udev_rules = 0/' \
+        -e 's/obtain_device_list_from_udev = 1/obtain_device_list_from_udev = 0/' \
+        /etc/lvm/lvm.conf
```
Comment (safronovD) [inline pkg/node/Dockerfile:13]: Could you please add some comments? I'm not really understand, what happens here :)

Comment (Aber4Nod) [reply pkg/node/Dockerfile]: Added the comments

File: pkg/base/linuxutils/partitionhelper/partition_helper.go [hunk 2]
```diff
@@ -175,8 +174,14 @@ func (p *WrapPartitionImpl) GetPartitionTableType(device string) (string, error)
 // CreatePartition creates partition with name partName on a device
 // Receives device path to create a partition
 // Returns error if something went wrong
-func (p *WrapPartitionImpl) CreatePartition(device, label string) error {
-	cmd := fmt.Sprintf(CreatePartitionCmdTmpl, device, label)
+func (p *WrapPartitionImpl) CreatePartition(device, label, partUUID string, setUUID bool) error {
+	var cmd string
+	switch setUUID {
```
Comment (rkiselenko) [inline pkg/base/linuxutils/partitionhelper/partition_helper.go:179]: Don't use `switch` for two statements, Use `if` here, please.

**Review by rkiselenko — APPROVED** [review]

File: pkg/base/linuxutils/partitionhelper/partition_helper.go (hunk 2 above)
Comment (Aber4Nod) [reply pkg/base/linuxutils/partitionhelper/partition_helper.go]: Refactored

File: pkg/node/Dockerfile.build [hunk 3]
```diff
@@ -5,4 +5,14 @@ ADD     health_probe    health_probe
 # Remove bash packet to get rid of related CVEs
 RUN     apt update --no-install-recommends -y -q; apt remove --no-install-recommends -y --allow-remove-essential -q bash; apt install --no-install-recommends -y -q util-linux parted xfsprogs lvm2 gdisk strace udev net-tools
 
-
+# Don't rely on udevd, it isn't available in some combinations of host/guest kernels.
```
Comment (mishoyama) [inline pkg/node/Dockerfile.build:8]: `isn't available` not precise term here. `not compatible` due to poll/epoll APIs sounds better. Please add md doc with your finding and post refence here

Comment (Aber4Nod) [reply pkg/node/Dockerfile.build]: Added docs, removed changes to Dockerfile due to after repeated tests the issue with lvm was not reproduced

**Review by yurkov-anton — APPROVED** [review]

**Review by mitsaa — APPROVED** [review]

**Review by mishoyama — APPROVED** [review]

**Pull request merged**